README.md

```diff
@@ -31,7 +31,76 @@ examples, tutorials, and an API reference.
 
 ## Use with JuMP
 
-Use DiffOpt with JuMP by following this brief example:
+### DiffOpt-JuMP API with `Parameters`
+
+```julia
+using JuMP, DiffOpt, HiGHS
+
+model = Model(
+    () -> DiffOpt.diff_optimizer(
+        HiGHS.Optimizer;
+        with_parametric_opt_interface = true,
+    ),
+)
+set_silent(model)
+
+p_val = 4.0
+pc_val = 2.0
+@variable(model, x)
+@variable(model, p in Parameter(p_val))
+@variable(model, pc in Parameter(pc_val))
+@constraint(model, cons, pc * x >= 3 * p) #??? InvalidConstraintRef TODO
+@objective(model, Min, 2x)
+optimize!(model)
+@show value(x) == 3 * p_val / pc_val
+
+# the function is
+# x(p, pc) = 3p / pc
+# hence,
+# dx/dp = 3 / pc
+# dx/dpc = -3p / pc^2
+
+# First, try forward mode AD
+
+# differentiate w.r.t. p
+direction_p = 3.0
+MOI.set(model, DiffOpt.ForwardConstraintSet(), ParameterRef(p), direction_p)
```

Comment:
`3.0` isn't really a set so I would rather call it `ForwardParametricValue`. Alternatively, we pass a `MOI.Parameter(direction_p)` but then it kind of means we plan to support passing `MOI.EqualTo` for constraints as well etc... in the future

Reply:
I agree,
I think the alternatives are:
```julia
MOI.set(model, DiffOpt.ForwardConstraintSet(), ParameterRef(p), Parameter(direction_p))
```
and
```julia
MOI.set(model, DiffOpt. ForwardParametricValue(), p, direction_p)
```

The first is more verbose, but it is consistent with the current API. In a follow-up PR I would do the `set_forward(...)` stuff so we have a JuMP-like API, besides the MOI-like API.

I think future support of EqualTo and friends would be nice. It is especially useful for variable bounds.

Reply:
Yes, so it makes sense to go with the first one, the users wanting to spare keystrokes will use the JuMP API anyway